ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -80,12 +82,13 @@ public static boolean isPseudoColumn(final ColumnName columnName) {
   }
 
   public static Set<ColumnName> pseudoColumnNames(final int pseudoColumnVersion) {
-    if (!PseudoColumns.PSEUDO_COLUMN_NAMES_BY_VERSION.containsKey(pseudoColumnVersion)) {
-      throw new KsqlException("Provided pseudoColumnVersion has no corresponding columns defined");
-    }
-    return PseudoColumns.PSEUDO_COLUMN_NAMES_BY_VERSION.get(pseudoColumnVersion);
+    return PseudoColumns.getPseudoColumnNamesByVersion(pseudoColumnVersion);
   }
 
+  @SuppressFBWarnings(
+      value = "MS_EXPOSE_REP",
+      justification = "PSEUDO_COLUMN_NAMES is ImmutableSet"
```

Comment:
Is the returned set of pseudocolumn names really an ImmutableSet? Looks like it's just a HashSet, unless I'm misreading the code.

Reply:
It is, this is Matthias' code that I sort of copy-pasted where I thought it should go during a merge conflict. Should I just remove this as I am exposing a mutable set?

Reply:
You can try removing it but I think findbugs is going to fail (that's the source of the warning that's being suppressed). We should fix the issue so that findbugs does not fail without the suppression.